src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -30,12 +31,12 @@ internal static void BindAttributeTypes(ImmutableArray<Binder> binders, Immutabl
             Debug.Assert(attributesToBind.Any());
             Debug.Assert((object)ownerSymbol != null);
```

Comment:
Why is this even a parameter to the method?

Reply:
Honestly? No idea, but I think that refactoring is out of scope for a nullable annotation pr.

---
In reply to: [367115354](https://github.com/dotnet/roslyn/pull/40946#discussion_r367115354) [](ancestors = 367115354)

Reply:
I agree it's out of scope. I just noticed and couldn't hold back my comment.